frontend/app/src/data/defaults.ts

```diff
@@ -80,40 +52,3 @@ export const TRADE_LOCATION_ARBITRUM_ONE = 'arbitrum one';
 export const TRADE_LOCATION_BASE = 'base';
 
 export const TRADE_LOCATION_GNOSIS = 'gnosis';
```

Comment:
any reason why do we still need optimism -> gnosis? do we use the values for exceptions anywhere?